common/test/acceptance/pages/studio/utils.py

```diff
@@ -160,6 +160,18 @@ def get_codemirror_value(page, index=0, find_prefix="$"):
     )
 
 
+def get_input_value(page, css_selector):
+    """
+    Returns the value of input field.
+    """
+    page.wait_for_element_presence(
+        css_selector,
+        'Elements matching "{}" selector are present'.format(css_selector)
+    )
+    selected_elements = page.q(css=css_selector)
+    return [selected_element.get_attribute('value') for selected_element in selected_elements]
```

Comment:
As written this method returns a list of the values for fields matching the css selector. However in all usages, the tests only use the first element in the list anyhow.

It would be better to instead simply return the value of the first element that matches. If so, please remember to also change the docstring to reflect this change.

Another thing, you should be able to use the built-in BrowserQuery.attrs method. 
See http://bok-choy.readthedocs.io/en/latest/_modules/bok_choy/query.html#BrowserQuery.attrs

Something like this (I haven't checked the syntax):
```
return page.q(css=css_selector).attrs('value')[0]
```